k8s/hammerdb/mariadb-conf/etc-mysql-my.cnf

```diff
@@ -105,11 +105,12 @@ long_query_time = 10
 
 # Performance schema enablement
 # see (https://mariadb.com/kb/en/performance-schema-overview/)
-performance_schema=ON
-performance-schema-instrument='stage/%=ON'
-performance-schema-consumer-events-stages-current=ON
-performance-schema-consumer-events-stages-history=ON
-performance-schema-consumer-events-stages-history-long=ON
+performance-schema=OFF
```

Comment:
fwiw, you also changed the `-` to `_` in this instance. Everything else has `-` not `_` so I don't what's correct, but wanted to point out the change.

Reply:
Done.